src/Drupal/Commands/core/LocaleCommands.php

```diff
@@ -128,4 +157,185 @@ public function update($options = ['langcodes' => self::REQ])
 
         drush_backend_batch_process();
     }
+
+    /**
+     * Exports to a gettext translation file.
+     *
+     * @see \Drupal\locale\Form\ExportForm::submitForm
```

Comment:
Lets comment this out because its leads to an error during docs generation  https://github.com/FriendsOfPHP/Sami/issues